### PR TITLE
Update requirements.txt and README

### PR DESCRIPTION
**Pull request by harshdattani**
NLTK version increase [#509](https://github.com/coala-analyzer/coala-bears/issues/509)

### CODE REVIEW

**Comment by harshdattani** [comment]
#509 and #481

**Comment by sils1297** [comment]
hey, could you take a look at http://coala.readthedocs.io/en/latest/Getting_Involved/Writing_Good_Commits.html#how-to-write-good-commit-messages ?

For the first commit you'll want to have e.g. "requirements.txt: Increase NLTK version" which holds more precise information in the shortlog. Also you'll want to have a `Closes <full issue url>` in the message body.

**Comment by sils1297** [comment]
(you can use `git rebase -i` to batch work with commits FWIW)

File: requirements.txt [hunk 1]
```diff
@@ -24,7 +24,7 @@ cmakelint==1.*
 vim-vint==0.3.*
 # Stable version 3.2 is problematic in Windows.
 # Use >=3.3,==3.* once nltk-3.3 is released.
-nltk==3.1.*
+nltk==3.2.*
```
Comment (sils1297) [inline requirements.txt]: did you read the comment in line 25? Sounds like we don't want this

**Comment by sils1297** [comment]
both https://github.com/coala-analyzer/coala-bears/pull/531/commits/692f85a0c2a526618d281b9f26d55a223f38bec3 and https://github.com/coala-analyzer/coala-bears/pull/531/commits/39d7abc3a1f1cb14944d31da410e95ff16498498 needs work

**Comment by harshdattani** [comment]
@sils1297 Thanks! Got it!

**Comment by Adrianzatreanu** [comment]
@harshdattani ping :D

**Comment by sils1297** [comment]
Hey? Still aroudn?

**Comment by AbdealiJK** [comment]
As this was unresponsive for a while, it has been moved to two different PRs https://github.com/coala-analyzer/coala-bears/pull/752 and https://github.com/coala-analyzer/coala-bears/issues/713

Closing this

**Pull request closed without merging**